README.md

```diff
@@ -7,916 +7,62 @@
 
 _Take control of your internal daemons!_
 
-**Pebble** helps you to orchestrate a set of local service processes as an organized set.
-It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can
-easily manage non-system processes independently from the system services, but it was
-designed with unique features that help with more specific use cases.
+**Pebble** is a lightweight Linux service manager that helps you orchestrate a set of local service processes as an organized set. It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
 
-  - [General model](#general-model)
-  - [Layer configuration examples](#layer-configuration-examples)
-  - [Using Pebble](#using-pebble)
-  - [Container usage](#container-usage)
-  - [Layer specification](#layer-specification)
-  - [API and clients](#api-and-clients)
-  - [Hacking / Development](#hacking--development)
-  - [Contributing](#contributing)
+Pebble's key features:
 
-## General model
+- Service management with [layers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)
+- Service [dependencies](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/service-dependencies/)
+- Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/logs/) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
+- [Health checks](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/health-checks/)
+- [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/)
+- Identities
+- Can be used in Machines and containers
```

Comment:
```suggestion
- Can be used in machines and containers
```

More generally: "machines and containers" feels very Juju-y language. For someone unfamiliar with our systems, maybe it's more something like "used on bare metal, VMs, and containers"?